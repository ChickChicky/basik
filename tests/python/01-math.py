print(905 + 64465)
print(64651 - 9596)
print(9596 - 64651)
print(6464 * 64654)
print(6546 / 30)
print(-0)
print(-950546)