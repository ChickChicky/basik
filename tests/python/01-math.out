65370
55055
-55055
417923456
218
0
-950546
